### ISSUE TITLE
Session Stops Pinging Server When User is Active

### ISSUE THREAD

**Issue by CaitlinLiveware** [LiveappSolutions/Liveapp#91]
**I'm submitting a ...**
- [ X] bug report
- [ ] feature request
- [ ] other <!

**Current behavior:**
User is active in the application however session expires and they are kicked out of the application.

**Expected / new behavior:**
User is active in the application and does not get kicked out of the application.

**Minimal reproduction of the problem with instructions:**
Appears to be random. 

**Liveapp version:** 5.61

**Browser:**  Chrome XX| Mobile Chrome XX 

**Anything else:**
Session Event Log reason when the above occurs is 'Session stopped Pinging the Server'.

**Comment by adamLiveware** [comment]
@paulfisher53 in an email you said:

> I could make the Ping Timeout a setting so you have more control over it

We would like that to be a setting built on Liveapp 5.61 if possible. We are working to upgrade to the latest but it will take some time, if we can get the setting in 5.61 then we will be able to fix a lot of problems right away.

**Comment by paulfisher53** [comment]
@adamLiveware @CaitlinLiveware 

Please merge the below fix with your `bin` folder. Once loaded, you can control the ping timeout with a new setting in your `web.config`.

```xml
<setting name="PING_TIMEOUT_MINS" serializeAs="String">
 <value>20</value>
</setting>
```
I made sure to use version 5.6.1 as you requested.

[Fix-91-v5.6.1.zip](https://github.com/ZoomApps/Liveapp/files/3246164/Fix-91-v5.6.1.zip)

**Comment by paulfisher53** [comment]
**Fix/mod is available for this issue**

We are closing this issue as a fix/mod has been supplied. 

**NOTE:** If the issue persists, please re-open the issue with the appropriate testing information.
